src/App.js

```diff
@@ -1,10 +1,18 @@
 import React from "react";
 import "./App.css";
 // STEP 4 - import the button and display components
-// Don't forget to import any extra css/scss files you build into the correct component
+// Don't forget to import any extra css/scss files you build into 
+// the correct component
+import ButtonComponents from './components/ButtonComponents/NumberButtons/Numbers';
+import DisplayComponents from './components/DisplayComponents/Display';
+import OperatorButtons from './components/ButtonComponents/OperatorButtons/Operators';
+import SpecialButtons from './components/ButtonComponents/SpecialButtons/Specials';
+
 
 // Logo has already been provided for you. Do the same for the remaining components
 import Logo from "./components/DisplayComponents/Logo";
+import OperatorButton from "./components/ButtonComponents/OperatorButtons/OperatorButton";
+import SpecialButton from "./components/ButtonComponents/SpecialButtons/SpecialButton";
```

Comment:
you wouldn't be needing to import these here. consider removing the lines since you did not use them.